FS.common.js

```diff
@@ -159,6 +161,8 @@ function readFileGeneric(filepath: string, encodingOrOptions:?string, command: F
 function readDirGeneric(dirpath: string, command: Function) {
   return command(normalizeFilePath(dirpath)).then(files => {
     return files.map(file => ({
+      created: file.created && new Date(file.created),
```

Comment:
Thank you for contributing! I think it is not a good idea to deliver the current date on Android (as file.created will not be available). We should instead deliver "null" here to indicate that this attribute simply not exists. Can you change this that way? i will merge then. thx :)